src/ui/wxWidgets/wxUtilities.cpp

```diff
@@ -269,6 +274,31 @@ wxBitmap wxUtilities::GetBitmapResource( const wxString& name )
     wxBitmap bitmap(cpane_xpm);
     return bitmap;
   }
+  else if (name == wxT("graphics/checkmark_placeholder.xpm"))
```

Comment:
Can we use a switch statement instead?

Reply:
I don't think C++ supports this for strings.

Reply:
That's a bummer. It works with enums though, so another option would be to change the `GetBitmapResource`'s argument to be an enum. That would also provide a nice compile-time check against unsupported resource names. Up to you.

Reply:
While working on the redesign of various dialogs, I noticed that there is a method called GetBitmapResource in the source code of every dialog implementation. This expects a path to an image file as a string and returns an image object. When the eye icon was introduced, this method had to be extended in several dialogs to take the path of the eye icon into account. These dialogs also had to get the same include statement for the eye icon. To avoid this redundancy, GetBitmapResource was introduced in wxUtilities. This function still follows the old concept that was introduced by a UI designer for the individual dialogs in order to keep the changes to a minimum. I fully agree that this could now be solved better in terms of programming. However, I would suggest considering this as a separate task, as a concept that is common to all dialogs and image resources should be introduced. This PR should keep its focus on the redesign of the Add-Edit-Entry dialog.